src/shared/ui/CharacterCounter.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { useTranslation } from 'react-i18next';
+
+import { Text } from '@shared/ui';
+
+import { colors } from '../lib';
+
+type Props = {
+  limit: number | string;
+  numberOfCharacters: number | string;
+  fontSize?: number;
+  focused?: boolean;
+};
+
+const CharacterCounter: FC<Props> = ({
+  numberOfCharacters,
+  limit,
+  fontSize = 14,
+  focused = false,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Text
+      style={[
+        styles.CharacterCounterText,
+        {
+          fontSize,
+          color: focused ? colors.primary : colors.grey4,
+        },
+      ]}
```

Comment:
Please avoid using inline styles for Tamagui's components unless you have a good reason